lib/code_climate/test_reporter/formatter.rb

```diff
@@ -15,9 +15,11 @@ class Formatter
       def format(result)
         return true unless CodeClimate::TestReporter.run?
 
-        print "Coverage = #{result.source_files.covered_percent.round(2)}%. "
+        simplecov_result = SimpleCov::Result.new(result.first[1]["coverage"])
```

Comment:
Hmm. This is a fun little bikeshed. I might use exception handling:

```rb
  result.fetch(0).fetch(1).fetch("coverage")
rescue NoMethodError, KeyError => ex
  raise InvalidSimpleCovResultError, ex.message
end
```

WDYT of something along those lines?